tests/utils.py

```diff
@@ -120,3 +77,36 @@ def dict_from_cookie(obj):
             driver.add_cookie(dict_from_cookie(cookie))
         except (UnableToSetCookieException, InvalidCookieDomainException):
             print(cookie)
+
+
+def _to_json(val):
+    def __to_serializable(val):
+        if isinstance(val, ModuleType):
+            return val.__name__
+        elif attr.has(val.__class__):
+            obj = _cleaned_dict_from_attrs(val)
+            if obj:
+                return obj
+            return str(val)
+        return _to_serializable(val, with_attrs=False)
+
+    return json.dumps(val, default=__to_serializable, sort_keys=True)
+
+
+def parametrize_ids(parameters_ids):
```

Comment:
Spent few minutes trying to guess how this thing works  (without knowing pytest parametrize ids api)
I think something like this would be more pythonic:

```
def parametrize_ids(parameters_ids):
    ids = parameters_ids.split(",")
    idgen = iter(ids)
    res = {}

    def wrap(value):
        res[next(idgen)] = value
        if len(res) == len(ids):
            return _to_json(res)

    return wrap
```

Reply:
@ikseek this solution doesn't work for example with next test test_batch_no_api_key_error